packages/collector/test/tracing/opentelemetry/test.js

```diff
@@ -53,18 +56,35 @@ mochaSuiteFn('opentelemetry tests', function () {
         if (process.env.INSTANA_TEST_SKIP_INSTALLING_DEPS === 'true') {
           return;
         }
-        execSync('npm install --no-save --no-package-lock --prefix ./ ./core.tgz', {
+
+        execSync('npm install --save --prefix ./ ./core.tgz', {
+          cwd: __dirname,
+          stdio: 'inherit'
+        });
+
+        execSync('npm install --save --prefix ./ ./collector.tgz', {
+          cwd: __dirname,
+          stdio: 'inherit'
+        });
+
+        execSync('npm install --save --prefix ./ @opentelemetry/api@1.9.0', {
           cwd: __dirname,
           stdio: 'inherit'
         });
 
-        execSync('npm install --no-save --no-package-lock --prefix ./ ./collector.tgz', {
+        execSync('npm install --save --prefix ./ "@opentelemetry/api-v1.3.0@npm:@opentelemetry/api@1.3.0"', {
           cwd: __dirname,
           stdio: 'inherit'
         });
       });
+
       // TODO: Restify test is broken in v24. See Issue: https://github.com/restify/node-restify/issues/1984
-      const restifyTest = semver.gte(process.versions.node, '24.0.0') ? describe.skip : describe;
+      let restifyTest = semver.gte(process.versions.node, '24.0.0') ? describe.skip : describe;
+
+      if (process.env.RUN_ESM === 'true') {
```

Comment:
Need to manually skip them because currently we don't have a strategy when multiple describes/apps are in one test file and some of them have esm support and some not. Just skipping them manually for now (not ideal).